code/modules/antagonists/spy/spy.dm

```diff
@@ -130,6 +130,25 @@
 		your_mission.owner = owner
 		your_mission.explanation_text = pick_list_replacements(SPY_OBJECTIVE_FILE, "objective_body")
 		objectives += your_mission
+	
+	if(prob(25))
```

Comment:
If this 25% rolls, the random 1-3 custom objectives should be changed to 0-2, otherwise this opens up spies getting 5 objectives? A tad much

Reply:
Made it so it'll only roll the protect options if the customs haven't filled up, removed their fail-condition from printing.